### PR TITLE
uploads

### PR DESCRIPTION
**Pull request by Danuragtiwari**

### CODE REVIEW

**Comment by Danuragtiwari** [comment]
merge it!!

**Review by exu3 — CHANGES_REQUESTED** [review]
Hey @Danuragtiwari! Could you add this dino to the README.md? Thanks!

**Comment by Danuragtiwari** [comment]
Hey, @exu3 now check it! If all fix then merge it otherwise ping me for the same.

**Review by exu3 — APPROVED** [review]
Nice work!

File: README.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ _The Dinosaur Who Can Never Look Back_
 
 ![](Wishy_S_dino.png)
 
+"slim_shady"
```
Comment (exu3) [inline README.md:55]:
```suggestion
"slim_shady"

```

**Pull request merged**